libs/service-portal/petitions/README.md

```diff
@@ -0,0 +1,9 @@
+<!-- gitbook-ignore -->
+
+# Service Portal Petitions
+
+This module displays what petitions are assigned to you, and allows you to un-register your petitions or register new petitions
+
+## Running unit tests
+
+Run `nx test service-portal-petitions` to execute the unit tests via [Jest](https://jestjs.io).
```

Comment:
Maybe add to readme that you need to run endorsement service to test locally?

Reply:
Lets do it when er overhaul the whole documentation for the endorsement system :)